openstack/migrate-helpers/osmigrator.rb

```diff
@@ -108,15 +108,15 @@ def secgroup_export
         importer_script.puts("# Start of #{secgroup_name} Rules")
 
         # Add the command to create the security group.
-        sec_description = group_details[0]["Value"]
+        sec_description = group_details["description"]
         if sec_description.nil?
           importer_script.puts("neutron security-group-create #{secgroup_name}")
         else
           importer_script.puts("neutron security-group-create #{secgroup_name} --description \"#{sec_description}\"")
         end
 
         # Parse out the actual rules. It requires a little bit of hackery because its not actually json.
-        rules = JSON.parse('[' + group_details[3]["Value"].gsub(/}\n/, "},") + ']')
+        rules = JSON.parse('[' + group_details["security_group_rules"].gsub(/}\n/, "},") + ']')
```

Comment:
Heh, whatever format they return here is absolutely hilarious. I tried to come up with a better way to do this and came to the same exact solution that you did.

Oh openstack CLIs...